### ISSUE TITLE
e.isShiftKlick not working on Spigot 1.15.2

### ISSUE THREAD

**Issue by Tanju42** [MinusKube/SmartInvs#136]
I`ve did a System out with all possible e.is* and this is the result:
Left
Left: true Right: false Shift: false
Right
Left: false Right: true Shift: false
Left Shift
Nothing in Console
Right Shift
Nothing in Console

This is how I did it:

```
ClickableItem ClickItem = ClickableItem.of(item, e -> {
            System.out.println("Left: "+e.isLeftClick()+" Right: "+e.isRightClick()+" Shift: "+e.isShiftClick() );
            if (e.isLeftClick()) {
                if (e.isShiftClick()) {
                    volume = volume + 0.1f;
                } else {
                    volume = volume + 1f;
                }
                if (volume != 99.9f) {
                    volume++;
                }
            } else if (e.isRightClick()) {
                if (e.isShiftClick()) {
                    volume = volume - 0.1f;
                } else {
                    volume = volume - 1f;
                }
                if (volume != 0.1f) {
                    volume--;
                }
            }
            SoundsInv.open(p, Page);
        });
```

**Comment by Tanju42** [comment]
When I do Shift+Left the Above code(System.out.println();) will not run.

So I think there is a problem in the code from SmartInvs

**Comment by Jamiezi** [comment]
I am also experiencing this issue, and I have also tried using InventoryActions to catch a shift click, but it seems that shift clicks aren't being registered at all.

**Comment by Tanju42** [comment]
For someone who needs the shift click

I created my own fork: https://github.com/Tanju42/smart-invs-fork

And here you can get the repo: https://jitpack.io/#Tanju42/smart-invs-fork/1.2.9


There I fixed the code

**Comment by MinusKube** [comment]
Fixed by #185
